.github/workflows/npm-beta.yml

```
@@ -118,49 +118,93 @@ jobs:
+          if [ "$PACKAGE_EXISTS" == "false" ]; then
+            echo "First time publishing - using version from package.json"
+            NEW_VERSION="$CURRENT"
+            echo "version=$NEW_VERSION" >> $GITHUB_OUTPUT
+            echo "First publish version: $NEW_VERSION"
+            exit 0
+          fi
+          
+          # Get all published versions from NPM
+          echo "Fetching published versions from NPM..."
+          NPM_VERSIONS=$(npm view @mikerob/lodgify-mcp versions --json 2>/dev/null || echo "[]")
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)